README.md

```diff
@@ -148,22 +156,20 @@ claude mcp list
 claude mcp remove zen -s user
 
 # You may need to remove an older version of this MCP after it was renamed:
-claude mcp remove gemini -s user
+claude mcp remove zen -s user
 ```
 Now run `claude` on the terminal for it to connect to the newly added mcp server. If you were already running a `claude` code session,
 please exit and start a new session.
 
-#### If Setting up for Claude Desktop
-
-- Open Claude Desktop
-- Go to **Settings** → **Developer** → **Edit Config**
+#### Option A: Local Development Setup (using local Docker build)
 
-This will open a folder revealing `claude_desktop_config.json`.
+If you ran `./setup-docker.sh`, it provided you with the exact configuration for local development.
 
-2. ** Update Docker Configuration**
+**Steps:**
+1. **Open Claude Desktop** → **Settings** → **Developer** → **Edit Config**
+   - This opens a folder revealing `claude_desktop_config.json`
 
-The setup script shows you the exact configuration. It looks like this. When you ran `setup-docker.sh` it should
-have produced a configuration for you to copy:
+2. **Add the configuration** shown by setup script (or copy from below):
```

Comment:
These are old instructions, headings are also old - I would say take the current README as is as the base, and kindly re-apply a few changes by refactoring and splitting it into various files, otherwise we will lose a lot of the improvements made to the README recently